src/main/java/org/jenkinsci/plugins/workflow/support/visualization/table/ArgumentsColumn.java

```diff
@@ -39,7 +40,7 @@
     @DataBoundConstructor public ArgumentsColumn() {}
 
     public String get(FlowNode node) {
-        return ArgumentsAction.getStepArgumentsAsString(node);
+        return StringUtils.substring(ArgumentsAction.getStepArgumentsAsString(node), 0, 80);
```

Comment:
Maybe it would be better to truncate to 77, and if the text was truncated, add `...` to make it clear that the content was truncated?

Reply:
sure, no problem